src/main/java/com/faforever/client/rankedmatch/MatchmakerInfoMessage.java

```diff
@@ -4,86 +4,47 @@
 import com.faforever.client.remote.domain.FafServerMessageType;
 import com.faforever.client.remote.domain.RatingRange;
 import com.google.gson.annotations.SerializedName;
+import lombok.Data;
+import lombok.Getter;
+import lombok.Setter;
 
 import java.util.List;
 
 public class MatchmakerInfoMessage extends FafServerMessage {
 
+  @Data
   public static class MatchmakerQueue {
 
-    private QueueName queueName;
+    private String queueName;
     private String queuePopTime;
+    @SerializedName("team_size")
+    private int teamSize;
+    @SerializedName("num_players")
+    private int numPlayers;
+
+    // The boundaries indicate the ranges applicable for other searching players,
+    // boundarys.size() therefore indicates the players currently in queue
     @SerializedName("boundary_75s")
     private List<RatingRange> boundary75s;
     @SerializedName("boundary_80s")
     private List<RatingRange> boundary80s;
 
-    public static enum QueueName {
-      @SerializedName("ladder1v1")
-      LADDER_1V1
-    }
-
-    public MatchmakerQueue(QueueName queueName, String queuePopTime, List<RatingRange> boundary75s, List<RatingRange> boundary80s) {
+    public MatchmakerQueue(String queueName, String queuePopTime, int teamSize, int numPlayers, List<RatingRange> boundary75s, List<RatingRange> boundary80s) {
       this.queueName = queueName;
       this.queuePopTime = queuePopTime;
+      this.teamSize = teamSize;
+      this.numPlayers = numPlayers;
       this.boundary75s = boundary75s;
       this.boundary80s = boundary80s;
     }
 
-    public QueueName getQueueName() {
-      return queueName;
-    }
-
-    public void setQueueName(QueueName queueName) {
-      this.queueName = queueName;
-    }
-
-    public String getQueuePopTime() {
-      return queuePopTime;
-    }
-
-    public void setQueuePopTime(String queuePopTime) {
-      this.queuePopTime = queuePopTime;
-    }
-
-    public List<RatingRange> getBoundary75s() {
-      return boundary75s;
-    }
-
-    public void setBoundary75s(List<RatingRange> boundary75s) {
-      this.boundary75s = boundary75s;
-    }
-
-    public List<RatingRange> getBoundary80s() {
-      return boundary80s;
-    }
-
-    public void setBoundary80s(List<RatingRange> boundary80s) {
-      this.boundary80s = boundary80s;
-    }
   }
-
-  private String action;
+  @Getter
+  @Setter
```

Comment:
I believe these can be removed with the @Data at top

Reply:
The @Data annotation is only for the nested class